PyRDF/backend/Dist.py

```diff
@@ -353,7 +353,7 @@ def _get_friend_info(self, tree):
 
         return FriendInfo(friend_names, friend_file_names)
 
-    def execute(self, generator):
+    def execute(self, generator, trigger_loop=True):
```

Comment:
distributed info operations won't work now, will they? Similar to how `Count` doesn't work in spark, it returns a fundamental type so we can't merge it. We should add all info operations to the operations not supported by `Dist`